#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_build-with-make-kpkg.dpatch by Ian Campbell <ijc@hellion.org.uk>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Build system changes for compatibility with make-kpkg and module 
## DP: assistant

@DPATCH@

--- ivtv-0.6.2.orig/i2c-drivers/Makefile
+++ ivtv-0.6.2/i2c-drivers/Makefile
@@ -21,7 +21,7 @@
 	$(MAKE) -C $(KDIR) M=$(CURDIR) modules
 
 install:: all
-	$(MAKE) INSTALL_MOD_PATH=$(DESTDIR) INSTALL_MOD_DIR=$(MDIR) \
+	$(MAKE) INSTALL_MOD_PATH=$(DESTDIR) INSTALL_MOD_DIR=$(MDIR) KERNELRELEASE=$(KVER) \
 		-C $(KDIR) M=$(CURDIR) modules_install
 
 clean::
--- ivtv-0.6.2.orig/driver/Makefile
+++ ivtv-0.6.2/driver/Makefile
@@ -20,10 +22,10 @@
 else
 
 all::
-	$(MAKE) -C $(KDIR) M=$(CURDIR) modules
+	$(MAKE) -C $(KDIR) M=$(CURDIR) KERNELRELEASE=$(KVER) modules
 
 install:: all
-	$(MAKE) INSTALL_MOD_PATH=$(DESTDIR) INSTALL_MOD_DIR=$(MDIR) \
+	$(MAKE) INSTALL_MOD_PATH=$(DESTDIR) INSTALL_MOD_DIR=$(MDIR) KERNELRELEASE=$(KVER) \
 		-C $(KDIR) M=$(CURDIR) modules_install
 
 clean::
